dashboard/app/controllers/sections_controller.rb

```diff
@@ -7,8 +7,7 @@ def show
     @secret_pictures = SecretPicture.all.shuffle
   end
 
-  # Allows you to update a section's course_id. Clears any assigned script_id
-  # in the process
+  # Allows you to update a section. Clears any assigned script_id in the process
   def update
```

Comment:
I could only find one existing caller of this function: https://github.com/code-dot-org/code-dot-org/blob/83df657f80ca88f4cacfb135688baa0d747f3cf9/apps/src/templates/courseOverview/AssignToSection.js#L52

Just mentioning since it was kind of tricky to track down. I don't see any reason your code changes would cause a problem there.